atom/browser/api/atom_api_system_preferences.h

```diff
@@ -93,6 +100,10 @@ class SystemPreferences : public mate::EventEmitter<SystemPreferences> {
   HWND window_;
 
   std::string current_color_;
+
+  bool invertered_color_scheme_ = false;
+
+  std::unique_ptr<gfx::ScopedSysColorChangeListener> color_change_listener_;
```

Comment:
Since `color_change_listener_` is initialized in constructor, there is no need to put `gfx::ScopedSysColorChangeListener` in a `unique_ptr`.